soco/services.py

```diff
@@ -858,6 +858,22 @@ def __init__(self, soco):
         self.event_subscription_url = "/MediaRenderer/RenderingControl/Event"
         self.DEFAULT_ARGS.update({"InstanceID": 0})
 
+    def get_eq_variable(self, variable: str) -> int:
+        """Return the EQ value for given variable as an integer."""
+        response = self.GetEQ([("InstanceID", 0), ("EQType", variable)])
+
+        return int(response["CurrentValue"])
+
+    def set_eq_variable(self, variable: str, value: int):
+        """Set the EQ variable to given (integer) value."""
+        return self.SetEQ(
+            [
+                ("InstanceID", 0),
+                ("EQType", variable),
+                ("DesiredValue", value),
+            ]
+        )
```

Comment:
I was trying to write some tests for all the functionality using this and realized that it's a bit more complicated as these methods are themselves magicmocks inside `moco`...

I'll have to see how to go around that limitation, in case it is fine to introduce these helpers (as indicated as an open question in the PR description) to reduce code duplication.